README.md

```diff
@@ -0,0 +1,18 @@
+[![Python Discourse chat](https://img.shields.io/badge/Discourse-join_chat-brightgreen.svg)](https://discuss.python.org/)
+
+Issue tracker and relevant tools for CPython's workflow.
+
+## Core workflow tools
+
+| Name                                                                                            | Description                                                          | Issue tracker                                                            | Owner/Maintainer                                 |
+|-------------------------------------------------------------------------------------------------|----------------------------------------------------------------------|--------------------------------------------------------------------------|--------------------------------------------------|
+| [python/bedevere](https://github.com/python/bedevere)                                           | Bot to help identify missing information for CPython pull requests   | [GitHub](https://github.com/python/bedevere/issues)                      | [Brett Cannon](https://github.com/brettcannon)   |
+| [python/blurb](https://github.com/python/blurb)                                                 | `blurb add` on the command line                                      | [GitHub](https://github.com/python/blurb/issues)                         |                                                  |
+| [python/blurb_it](https://github.com/python/blurb_it)                                           | `blurb add` on the web                                               | [GitHub](https://github.com/python/blurb_it/issues)                      | [Mariatta](https://github.com/mariatta)          |
+| [python/cherry-picker](https://github.com/python/cherry-picker)                                 | Command line tool for backporting CPython pull requests              | [GitHub](https://github.com/python/cherry-picker/issues)                 | [Mariatta](https://github.com/mariatta)          |
+| [python/miss-islington](https://github.com/python/miss-islington)                               | Bot for backporting CPython pull requests                            | [GitHub](https://github.com/python/miss-islington/issues)                | [Mariatta](https://github.com/mariatta)          |
+| [ambv/cla-bot](https://github.com/ambv/cla-bot)                                                 | CLA enforcement bot for Python organization projects                 |                                                                          | [Łukasz Langa](https://github.com/ambv)          |
+| [berkerpeksag/cpython-emailer-webhook](https://github.com/berkerpeksag/cpython-emailer-webhook) | Webhook to send every CPython commit to python-checkins mailing list | [GitHub](https://github.com/berkerpeksag/cpython-emailer-webhook/issues) | [Berker Peksag](https://github.com/berkerpeksag) |
```

Comment:
I gave it a try -- it's not much better.
```suggestion
| Name | Description | Issue tracker | Owner/Maintainer |
|------|-------------|---------------|------------------|
| [python/bedevere](https://github.com/python/bedevere)  |  Bot to help identify missing information for CPython pull requests  |  [GitHub](https://github.com/python/bedevere/issues)  |  [Brett Cannon](https://github.com/brettcannon) |
| [python/blurb](https://github.com/python/blurb)  |  `blurb add` on the command line  |  [GitHub](https://github.com/python/blurb/issues)  |  |
| [python/blurb_it](https://github.com/python/blurb_it)  |  `blurb add` on the web  |  [GitHub](https://github.com/python/blurb_it/issues)  |  [Mariatta](https://github.com/mariatta) |
| [python/cherry-picker](https://github.com/python/cherry-picker)  |  Command line tool for backporting CPython pull requests  |  [GitHub](https://github.com/python/cherry-picker/issues)  |  [Mariatta](https://github.com/mariatta) |
| [python/miss-islington](https://github.com/python/miss-islington)  |  Bot for backporting CPython pull requests  |  [GitHub](https://github.com/python/miss-islington/issues)  |  [Mariatta](https://github.com/mariatta) |
| [ambv/cla-bot](https://github.com/ambv/cla-bot)  |  CLA enforcement bot for Python organization projects |  |  [Łukasz Langa](https://github.com/ambv) |
| [berkerpeksag/cpython-emailer-webhook](https://github.com/berkerpeksag/cpython-emailer-webhook)  |  Webhook to send every CPython commit to python-checkins mailing list  |  [GitHub](https://github.com/berkerpeksag/cpython-emailer-webhook/issues)  |  [Berker Peksag](https://github.com/berkerpeksag) |
```

Reply:
We can also use HTML in Markdown, how does the newest commit look?